docs/custom_modules.md

```diff
@@ -4,16 +4,74 @@
 
 The custom modules used for this project are:
 
-- [Drupal Contribution Tracker](https://github.com/contrib-tracker/backend/tree/main/web/modules/custom/ct_drupal): provides functionality for tracking contributions on Drupal.org
+### [Drupal Contribution Tracker](https://github.com/contrib-tracker/backend/tree/main/web/modules/custom/ct_drupal)
 
-- [GitHub Contribution Tracker](https://github.com/contrib-tracker/backend/tree/main/web/modules/custom/ct_github): provides functionality for tracking issues and PR contributions on GitHub.
+The Drupal.org Contribution Tracker module (ct_drupal) automates tracking user contributions made specifically on the Drupal.org platform. It achieves this by:
 
-- [Contribution Plugin Manager](https://github.com/contrib-tracker/backend/tree/main/web/modules/custom/ct_manager): provides the feature to create a custom Plugin to track and store contributions from different sources and send notifications on Slack Channel.
+- The module utilizes a `ContributionSource` plugin named `DrupalContribution.php`. This plugin interacts with the `Drupal.org API` via a wrapper module built around `Guzzle 6` for efficient communication.
 
-- [Contribution Tracker Reports](https://github.com/contrib-tracker/backend/tree/main/web/modules/custom/ct_reports): provides functionality to generate reports based on contribution data.
+- To track contributions, the module requires each user's Drupal.org username to be filled in a dedicated field (managed by the `do_username` dependency).
 
-- [Contrib Tracker Users](https://github.com/contrib-tracker/backend/tree/main/web/modules/custom/ct_user): manages user-login activities in Contrib Tracker.
+- During cron runs, the system fetches contributions for all users with a populated Drupal.org username field. It retrieves the latest 100 code contributions and 100 issues for each user.
```

Comment:
For this and also for `ct_github`, the entry point is that the cron job calls `ct_manager` which calls this process in all the plugins. So, this sentence is slightly misleading as someone reading it may think that we directly implement hook_cron in these modules.